### PR TITLE
add task057

### PR DESCRIPTION
**Pull request by eshenko**
Hello! Please check my new task! Thx

### CODE REVIEW

File: src/test/java/com/epam/university/java/core/task057/Task057Test.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    @Test
+    public void severalFiles() {
+        String path = getClass().getResource("/task057/").getPath();
+        String target1 = getClass().getResource("/task057/subdir1/test1.txt").getPath();
+        String target2 = getClass().getResource("/task057/subdir2/test2.txt").getPath();
+        String target3 = getClass().getResource("/task057/subdir2/subsubdir/test3.txt").getPath();
+        List<String> result = instance.find(path, "test3");
+        assertTrue("Incorrect list size or result isn't contain target",
```
Comment (aabarmin) [inline src/test/java/com/epam/university/java/core/task057/Task057Test.java]: It's better having separate assertions for every case

File: src/test/java/com/epam/university/java/core/task057/Task057Test.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        List<String> result = instance.find(path, "test2");
+        assertTrue("Incorrect list size or result isn't contain target",
+                result.size() == 1
+                        && result.contains(getClass().getResource("/task057/test.txt").getPath()));
+    }
+
+    @Test
+    public void severalFiles() {
+        String path = getClass().getResource("/task057/").getPath();
+        String target1 = getClass().getResource("/task057/subdir1/test1.txt").getPath();
```
Comment (aabarmin) [inline src/test/java/com/epam/university/java/core/task057/Task057Test.java]: I suppose, it's better using something like `Path.of()` and then `Path.resolve()` to join path's parts.

**Review by aabarmin — CHANGES_REQUESTED** [review]

File: src/test/java/com/epam/university/java/core/task057/Task057Test.java (hunk 1 above)
Comment (eshenko) [reply src/test/java/com/epam/university/java/core/task057/Task057Test.java]: Fixed! Please check

File: src/test/java/com/epam/university/java/core/task057/Task057Test.java (hunk 2 above)
Comment (eshenko) [reply src/test/java/com/epam/university/java/core/task057/Task057Test.java]: I think fixed:) Please check

**Comment by aabarmin** [comment]
Merged separately

**Pull request closed without merging**